src/arco_era5/update_co.py

```diff
@@ -0,0 +1,125 @@
+import apache_beam as beam
+import logging
+import subprocess
+import tempfile
+import os
+import datetime
+import xarray as xr
+import zarr as zr
+from dataclasses import dataclass
+from contextlib import contextmanager
+import calendar
+
+logger = logging.getLogger(__name__)
+
+variable_dict = {
```

Comment:
capitalize this as if it's a GLOBAL_CONSTANT.

Reply:
Added.